docs/resources/mysql.md

```diff
@@ -49,6 +49,7 @@ The following arguments are supported:
 * `subnet_no` - (Required) The ID of the associated Subnet. Public domain can only be used on a DB server generated on Public Subnet. 
 * `image_product_code` - (Optional) Image product code to determine the MySQL instance server image specification to create. If not entered, the instance is created for default value. It can be obtained through [`ncloud_mysql_image_products` data source](../data-sources/mysql_image_products.md)
 * `product_code` - (Optional) Product code to determine the MySQL instance server image specification to create. It can be obtained through [`ncloud_mysql_products` data source](../data-sources/mysql_products.md). Default : Minimum specifications(1 memory, 2 cpu)
+* `engine_version_code` - (Optional) MySQL engine version code. Only entered when `generation_code` is G3, If not entered, generate with the latest version currently available.
```

Comment:
MySQL engine version code. If not entered, generate with the default version currently available.